containers/Connection.ts

```diff
@@ -1,54 +1,73 @@
 import { createContainer } from "unstated-next";
 import { useState, useEffect } from "react";
 import { ethers } from "ethers";
+import { API as OnboardApi, Wallet } from "bnc-onboard/dist/src/interfaces";
+import { API as NotifyApi } from "bnc-notify/dist/src/interfaces";
+import Onboard from "bnc-onboard";
+import Notify from "bnc-notify";
 import { Observable } from "rxjs";
 import { debounceTime } from "rxjs/operators";
 
+import { config } from "./Config";
```

Comment:
If you wrap all the onboard related stuff into an `onboard` key you can then do `const { onboard } = config`,

and then easily access `dappId: onboard.apiKey`

Reply:
I don't think the `onboard` object should go in the config. it's more part of `Connection.ts`'s context, where it is right now. 

If we were to move it then it should perhaps go to a `onboard.ts` file that `Connection.ts` could import. This `onboard.js` would also use the `config.ts` file. However, I think this is an over-engineered solution and leaving it where it is now in `Connection.ts` is simple enough while still having sufficient separation of concerns.